### PR TITLE
Update New Features

### PR DESCRIPTION
**Pull request by rahulkumarsingh73690**
I added the new feature to take a screenshot in windows 10 after creator update.

### CODE REVIEW

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 1]
```diff
@@ -233,5 +233,6 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
 [Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.<br>
 [What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.<br>
 [What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.<br>
+[Advance Method to take screenshot in windows 10](https://goo.gl/FH2Xd4): See what's new feature for taking screenshot.<br>
```
Comment (illfated) [inline windows/whats-new/whats-new-windows-10-version-1803.md]: First of all, external links are unlikely to be included in Microsoft documentation.
Secondly, "Advance Method" should probably be "Advanced Method".
`See what's new feature for taking screenshot.` needs to be written in proper English.
The whole line needs a rewrite to be on par with Microsoft standards.

Comment (rahulkumarsingh73690) [reply windows/whats-new/whats-new-windows-10-version-1803.md]: Check i Updated as per your T & C

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 2]
```diff
@@ -233,5 +233,6 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
 [Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.<br>
 [What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.<br>
 [What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.<br>
+[Advanced Method to take screenshot in windows 10](https://goo.gl/FH2Xd4): See what's new features for taking screenshot.<br>
```
Comment (illfated) [inline windows/whats-new/whats-new-windows-10-version-1803.md]: Minor improvement, but still needs more. I think the last part should be different, something like:
`New features for taking screenshots.`

My major concern is the link, which points to https://www.rahulit.com/how-to-take-a-screenshot-on-a-dell-laptop/
Microsoft will not accept links in their documentation unless the linked page or content is owned by Microsoft.

**Comment by illfated** [comment]
Fair enough, I don't have anything more to add at the moment.
We just have to wait for a Microsoft Docs team member to comment.

**Comment by jdeckerms** [comment]
@rahulkumarsingh73690 Sorry, @illfated is correct - that isn't an appropriate link for this content library.

**Pull request closed without merging**